app/src/main/java/com/github/se/icebreakrr/ui/sections/Settings.kt

```diff
@@ -89,7 +93,12 @@ fun SettingsScreen(profilesViewModel: ProfilesViewModel, navigationActions: Navi
 
           Button(
               onClick = {
-                Toast.makeText(context, "Log Out (Not yet implemented)", Toast.LENGTH_SHORT).show()
+                if (isTesting) {
+                  navigationActions.navigateTo(Screen.AUTH)
+                } else {
+                  auth = FirebaseAuth.getInstance()
+                  auth.currentUser?.let { logout(context, navigationActions) }
+                }
               },
               colors = ButtonDefaults.buttonColors(containerColor = Color(0xFFCE0E00)),
```

Comment:
Hardcoded color, should be classified in Color.kt and use the constant from there for more maintanability

Reply:
I didn't work on Settings.kt but I can fix that sure